tests/behat/features/ra_management_grants.feature

```diff
@@ -0,0 +1,181 @@
+Feature: A RA(A) should only have access to certain pages
+
+  Scenario: Provision a institution and a user to promote later on by an authorized institution
+    Given institution "stepup.example.com" can "select_raa" from institution "stepup.example.com"
```

Comment:
This is the default right? If not set, then select_raa applies for that institution. Or at least it should be like this.

Reply:
We have a migration in MW which should take care of this (`20181001082254`). But as far as I know this is not implemented in our test fixture so we should push this manually. This was done because we couldn't detect the difference between null and empty in the authorization table.